downstream/modules/platform/con-controller-playbook-run-jobs.adoc

```diff
@@ -0,0 +1,25 @@
+[id="controller-playbook-run-jobs"]
+
+= Playbook run jobs
+
+When a playbook is executed, the full results automatically display in the *Output* tab. 
+This shows the same information you see if you run it through the Ansible command line, and can be useful for debugging.
```

Comment:
If used, the Ansible CLI displays the same information. This can be useful for debugging.